src/style.css

```diff
@@ -1 +1,17 @@
-/* styles go here */
+.body {
+  margin: 0;
+}
+
+.form {
+  margin-bottom: 20px;
+
+}
+
+.form-field {
+
+  padding-bottom: 10px;
+}
+
+.form-field:last-child {
+  padding-bottom: 0;
+}
```

Comment:
no needs to add all of these classes. 
You may only apply those to your `filedset` and form field (`label` or `div`)
Example: 
```
.form-fieldset:not(:last-child) {
  margin-bottom: 20px
}

.form-field:not(:last-child) {
  margin-bottom: 10px
}